packages/hardhat-test-helpers/src/helpers/getStorageAt.ts

```diff
@@ -0,0 +1,24 @@
+import type { NumberLike } from "../types";
+import {
+  getHardhatProvider,
+  assertValidAddress,
+  toRpcQuantity,
+} from "../utils";
+
+export async function getStorageAt(
+  hexAddress: string,
+  index: NumberLike,
+  block = "latest"
```

Comment:
I think this should be `block: NumberLike | "latest" = "latest"` instead, otherwise only strings can be passed as the block.